Documentation/TypoScript/Index.rst

```diff
@@ -0,0 +1,170 @@
+
+..  figure:: ActiveTypoScript.png
+    :alt: Screenshot of the submodule "Active TypoScript" in the module "Site Management > TypoScript"
+
+    Switch the submodules of module :guilabel:`Site Management > TypoScript`
+    using the Dropdown in the module header.
+
+You can find the variable `page` that you just defined. The other variables have
+been created by TypoScript loaded globally by the TYPO3 Core and system
+extensions. They are some of the
+:ref:`Reserved top-level objects <t3tsref:top-level-objects-other-reserved-tlo-s>`.
+
+..  _typoscript-play-around:
+
+Play around with TypoScript
+===========================
+
+You can now try out a couple of TypoScript commands to familiarize yourself
+with TypoScript.
+
+Here are some examples:
+
+..  _typoscript-example-stdwrap:
+
+Wrap "Hello, world." in p-tags
+------------------------------
+
+..  literalinclude:: _stdwrap.typoscript
+    :caption: config/sites/main/setup.typoscript
+    :linenos:
+    :emphasize-lines: 4-7
+
+4.  As we now have several options for :ref:`TEXT <t3tsref:cobj-text>` object
+    with path `page.10`, we switch to the
+    :ref:`block syntax <t3coreapi:typoscript-syntax-syntax-curly-brackets>` here.
+5.  Assign the text to the :confval:`value <t3tsref:text-value>` property of
+    the TEXT object.
+6.  We use the :confval:`stdWrap <t3tsref:text-stdwrap>` property of the TEXT
+    object to configure the :ref:`stdWrap <t3tsref:stdwrap>` function.
+
+    In this function we use option wrap :confval:`wrap <t3tsref:stdwrap-wrap>`.
+    It surrounds the current content of the TEXT object as set in line 5 with
+    the value defined here. The pipe `|` character is replaced with the text.
+
+..  note::
+    You may sometimes see that :confval:`stdWrap <t3tsref:text-stdwrap>`
+    functionality is directly applied to a TEXT object like this:
+
+    ..  literalinclude:: _stdwrap_direct.typoscript
+        :caption: config/sites/main/setup.typoscript
+        :linenos:
+        :emphasize-lines: 6
+
+    For backward compatibility reasons it is possible to apply stdWrap
+    properties directly to TEXT object. This is only true for TEXT objects, not
+    any other TypoScript cObject types. So we recommend to always use
+    stdWrap to stay consistent.
+
+..  _typoscript-example-page-title:
+
+Display the title of the current page on top
+--------------------------------------------
+
+..  literalinclude:: _page-title.typoscript
+    :caption: config/sites/main/setup.typoscript
+    :linenos:
+    :emphasize-lines: 3-7
+
+3.  We assign a second content object (also called "cObject") of type
+    :ref:`TEXT <t3tsref:cobj-text>` to
+    :confval:`index number 5 <t3tsref:page-array>` of `page`. As the index is
+    smaller then the index 10 of the TEXT object containing the text
+    "Hello World" it is displayed before the other object.
+4.  Use the :ref:`block syntax <t3coreapi:typoscript-syntax-syntax-curly-brackets>`
```

Comment:
```suggestion
4.  Uses the :ref:`block syntax <t3coreapi:typoscript-syntax-syntax-curly-brackets>`
```

Reply:
This is an interogation, "you should use" so I would keep my version